webapp/packages/core-blocks/src/FormControls/InputField.tsx

```diff
@@ -84,12 +85,14 @@ export const InputField: InputFieldType = observer(
       labelTooltip,
       mod,
       autoHide,
+      canShowPassword = true,
       onChange,
       onCustomCopy,
       ...rest
     }: ControlledProps | ObjectProps<any, any>,
-    ref: React.Ref<HTMLInputElement>,
+    ref,
   ) {
+    const inputRef = useRef<HTMLInputElement | null>(null);
```

Comment:
use useRefInherit instead (in case when you want to access ref inside component and forward it)